lib/web_ui/dev/build.dart

```diff
@@ -60,6 +60,8 @@ class BuildCommand extends Command<bool> {
       PipelineWatcher(
         dir: libPath.absolute,
         pipeline: buildPipeline,
+        // Ignore font files that are copied whenever tests run.
+        ignore: (event) => event.path.endsWith('.ttf'),
```

Comment:
Alternatively the copying logic could check if the file is the same and not copy anything. Either way works for me.